espanso-render/src/extension/shell.rs

```diff
@@ -31,6 +31,7 @@ use thiserror::Error;
 pub enum Shell {
   Cmd,
   Powershell,
+  Pwsh,
```

Comment:
Can I have a comment here indicating powershell v7+ and classic powershell?
or `Windows Powershell` and `Powershell` (core?) as it is named on the docs

Reply:
Sure, done in 362b0bc.